targets/TARGET_STM/TARGET_STM32F0/objects.h

```diff
@@ -22,6 +22,9 @@
 #include "PinNames.h"
 #include "gpio_object.h"
 
+#include "us_ticker_defines.h"
+#include "lp_ticker_defines.h"
```

Comment:
Eh? Why wasn't us_ticker_defines already there?

Oh, looks like this was broken in 47407759

So the `wait_us` optimisations have been broken for STM since Mbed OS 6.1, and the custom-tickers stuff would never have been doing anything.

This should be included in the documentation - you are also re-adding the missing USTICKER optimisations.

**But only for STM32F0** in this version. 

You should put the usticker include up in device.h, as it's a shared not-target-specific header, to reactivate it for all devices.

Reply:
@kjbracey-arm 
Updates done for all STM32